webapp/src/types/telemetry.ts

```diff
@@ -28,6 +28,8 @@ export enum PlaybookRunEventTarget {
     RequestUpdateClick = 'playbookrun_request_update',
     Participate = 'playbookrun_participate',
     Leave = 'playbookrun_leave',
+    AddParticipant = 'playbookrun_add_participant',
```

Comment:
I'd rename it to make more clear that accept multiple
```suggestion
    AddParticipant = 'playbookrun_add_participants',
```

Reply:
and same for server-side names